sources/Outputs/src/ParNetwork.cpp

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright (c) 2022, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
```

Comment:
I think this class is useless, either we keep the network one, either this one.
The evolution here is similar of what done for Solver.h .cpp class